src/store/storage-controller.ts

```diff
@@ -111,19 +109,53 @@ export class StorageController {
     return messages;
   }
 
-  public static async delete(
-    messageStore: MessageStore,
-    dataStore: DataStore,
+  public async delete(
     tenant: string,
     message: BaseMessage
   ): Promise<void> {
     const messageCid = await Message.getCid(message);
 
     if (message.descriptor.dataCid !== undefined) {
-      await dataStore.delete(tenant, messageCid, message.descriptor.dataCid);
+      await this.dataStore.delete(tenant, messageCid, message.descriptor.dataCid);
     }
 
-    await messageStore.delete(tenant, messageCid);
+    await this.messageStore.delete(tenant, messageCid);
+  }
+
+  public getMessage(tenant: string, messageCid: string): Promise<BaseMessage|undefined> {
+    return this.messageStore.get(tenant, messageCid);
+  }
+
+  public putMessage(
+    tenant: string,
+    messageJson: BaseMessage,
+    indexes: Record<string, string>,
+    options?: MessageStoreOptions): Promise<void> {
```

Comment:
i think we normally put the `): Promise<void> {` onto a separate line indented to match the line with the `(`

Reply:
Addressed in https://github.com/TBD54566975/dwn-sdk-js/pull/338/commits/931ab6bd4e5079229f6b9663de5a442eedfe4983